src/show/pages/public/Project/index.js

```diff
@@ -90,37 +54,42 @@ const Project = () => {
 
           <div className='mr-5 h-28 mt-5'>
             <div className='no-scrollbar flex ml-4 h-[112px] no-scrollbar overflow-x-scroll'>
-              {restPictures.map((item, index) => (
-                <img
-                  key={index}
-                  src={item}
-                  alt='bacpack'
-                  className='h-28 mx-2 rounded-lg'
-                />
-              ))}
+              {restPictures &&
+                restPictures.map((item, index) => (
+                  <img
+                    key={index}
+                    src={item}
+                    alt='bacpack'
```

Comment:
backpack is always the alt attribute? should be `cover-photo` or `main-photo` or `secondary-photo`.